### PR TITLE
Adds IPC plushie and toast, toast is made via IPC plushie.

### PR DESCRIPTION
**Pull request by Drakeven**
## What Does This PR Do
This PR adds an IPC plushie that can be hugged and loved, unlike the real ones. If you use a 'Bread Slice' on the plushie, it will produce toast with a satisfying ping. 

## Why It's Good For The Game
Vox have plushies of their own, @Cazdon suggested I make one for IPCs. The only issue I see with this PR is that so far, EMPS do not explode the plushie. Therefore, IPC mains may get jealous. 

I'm thinking of making some more race-plushies, so every race get some love. 

## Images of changes
7771dbf172d78e6da473d1536f2a03eb

## Changelog
:cl:
add: IPC plushie and toast.
/:cl:

### CODE REVIEW

**Comment by Cazdon** [comment]
This might make IPC's too overpowered, simply by associating them with an item that doesn't evaporate from EMPs.

I LOVE IT!

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1085,6 +1085,19 @@ obj/item/toy/cards/deck/syndicate/black
...
 	..()
 
+/obj/item/toy/plushie/ipcplushie
+	name = "ipc plushie"
+	desc = "An adorable ipc plushie, straight from New Canaan. Arguably more durable than the real deal. Toaster functionality included."
+	icon_state = "plushie_ipc"
+	item_state = "plushie_ipc"
+
+/obj/item/toy/plushie/ipcplushie/attackby(obj/item/B, mob/user, params)
+	if(istype(B, /obj/item/reagent_containers/food/snacks/breadslice))
```
Comment (Citinited) [inline code/game/objects/items/toys.dm]: This should `return ..()` if attacked by anything other than bread

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -1085,6 +1085,19 @@ obj/item/toy/cards/deck/syndicate/black
 		return
 	..()
 
+/obj/item/toy/plushie/ipcplushie
+	name = "ipc plushie"
+	desc = "An adorable ipc plushie, straight from New Canaan. Arguably more durable than the real deal. Toaster functionality included."
```
Comment (Citinited) [inline code/game/objects/items/toys.dm]: I think IPC should be capitalised here

**Comment by Carthusia** [comment]
I'm getting the song by Weird Al in my head, its called YEAH TOAST!

https://www.youtube.com/watch?v=c0NdwzdOLkE

Loven the plushies

**Comment by Cazdon** [comment]
> 
> 
> I'm getting the song by Heywood Banks in my head, its called YEAH TOAST!

😏 Might the reason behind the description of the toast?

**Comment by VeteranKyl** [comment]
Will the toast be usable in any other food crafting recipes? It'd be funny to see the chef using the plush as a way of making toast and jam/butter and such.

**Comment by Carthusia** [comment]
lol it just hit me, what if its not a plush at all, its literaly just a toaster that has googly eyes pasted onto it

**Comment by Drakeven** [comment]
Heh, yeah. YEAH TOAST was an insperation, I was listening to it while pring this.

**Comment by Couls** [comment]
make their monitor pop out when they're emped like those cartoon alarms
image

**Comment by pinatacolada** [comment]
Way farfetched but what if dropping the plushie in the pool makes it electrocute everyone in it

**Comment by Spacemanspark** [comment]
Emag the plushie for the electrocution effect :^)

**Comment by Citinited** [comment]
Emag it to fire the toast at the nearest mob

**Comment by Cazdon** [comment]
Oh oh! Make attacking it with a fork shock you, similar to a door. Mama always said not to stick forks into toasters.

File: code/game/objects/items/toys.dm (hunk 2 above)
Comment (Citinited) [reply code/game/objects/items/toys.dm]: Don't forget to capitalise IPC in the description too!

**Comment by rb303** [comment]
I recognise the head of that little cute thing.
The prototype was better.

**Comment by DogsParadise** [comment]
Here you go, a new ipc plushie sprite with animation. 4 frames, only face is animated. Kept the same idea that you wanted to incorporate into your sprite. Yes this sprite does look different than the other plushie sprites.

Ipc plushie

**Comment by Fox-McCloud** [comment]
Could ya solve the merge conflicts on this please?

**Comment by Drakeven** [comment]
Having issues pushing to this exact branch for some reason. Sorry I'm such a dumb. New branch can be found here, with no merge conflicts-  https://github.com/ParadiseSS13/Paradise/pull/13607

**Comment by Drakeven** [comment]
I realized what was wrong. Two capital letters didn't realise I was on a different branch. Please forgive my incompetence, oh mighty maintainers.

**Pull request merged**